packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -599,6 +603,10 @@ export const Table = <
     setPageSize(pageSize);
   }, [pageSize, setPageSize]);
 
+  const numSelectedRows =
+    selectionMode === 'single'
+      ? 0
```

Comment:
What if one row is selected when `selectionMode` is single? Then `numSelectedRows` should be 1?